src/main/js/Upload/FileUpload.tsx

```diff
@@ -217,15 +212,24 @@ class FileUpload extends React.Component<Props, State> {
             </Header>
           )}
           <Breadcrumb repository={repository} baseUrl={baseUrl} path={path} revision={revision} />
-          <FileMetaData path={path} changePath={this.changePath} />
+          <FileMetaData
+            path={path}
+            changePath={changedPath => {
+              this.setState({
+                path: changedPath,
+                shouldValidate: false
+              });
+            }}
```

Comment:
I like this solution, that a change is triggered only after a real change of the path value, and not after a focus event. 👍🏽